packages/fiori/src/ShellBar.ts

```diff
@@ -353,10 +398,15 @@ class ShellBar extends UI5Element {
 	_overflowPopoverExpanded = false;
 
 	@property({ type: Boolean, noAttribute: true })
-	_fullWidthSearch = false;
+	_isSBreakpoint = false;
 
 	@property({ type: Boolean, noAttribute: true })
-	_isXXLBreakpoint = false;
+	hasVisibleStartContent = false;
```

Comment:
I think it is designed to make use of the basic invalidation cycle in order to update the separators properly on every resize, since its visibility depends on css classes.